docs/gno-tooling/cli/gnokey/full-security-tx.md

```diff
@@ -0,0 +1,119 @@
+---
+id: full-security-tx
+---
+
+# Making an airgapped transaction
+
+`gnokey` provides a way to create a transaction, sign it, and later
+broadcast it to a chain in an airgapped manner. This approach, while more 
+complicated, grants users full control over the creation, signing, and broadcasting 
```

Comment:
grants full control and gives airgap support.

Reply:
[de91b1c](https://github.com/gnolang/gno/pull/2226/commits/de91b1c0006f77b8d1b81e165507a127a8bf39ee)